From: Jan Tojnar <jtojnar@gmail.com>
Date: Sat, 2 Oct 2021 00:13:14 +0200
Subject: =?utf-8?q?docs=3A_Fix_build_with_EDS_=E2=89=A5_3=2E41?=

Docs need the same flag as source code to be able to use fixes from https://gitlab.gnome.org/GNOME/folks/-/merge_requests/49.

(cherry picked from commit 62d588b0c609de17df5b4d1ebfbc67c456267efc)
---
 docs/meson.build | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/docs/meson.build b/docs/meson.build
index 7ace84b..05264f9 100644
--- a/docs/meson.build
+++ b/docs/meson.build
@@ -35,6 +35,10 @@ valadoc_targets = [
 ]
 
 if eds_backend_enabled
+  if eds_dep.version().version_compare('>=3.41')
+    docs_common_valadoc_flags += ['-D', 'HAS_EDS_3_41']
+  endif
+
   valadoc_targets += {
     'name': 'folks-eds',
     'source_files': eds_backendlib_sources,
